crates/sui-framework/packages/sui-framework/sources/priority_queue.move

```diff
@@ -69,7 +69,14 @@ module sui::priority_queue {
         };
         res
     }
-
+    //Return the length of queue
+    public fun length<T: drop>(pq: &PriorityQueue<T>): u64 {
+        pq.entries.length()
+    }
+    // Return `true` if the entries has no elements and `false` otherwise.
+    public fun is_empty<T: drop>(pq: &PriorityQueue<T>): bool {
+        pq.entries.length() == 0
```

Comment:
```suggestion
        pq.entries.is_empty()
```

Reply:
Thank you for your suggestion. It has been modified